ckks/bootstrapping/bootstrapper.go

```diff
@@ -232,7 +231,7 @@ func newBootstrapperBase(params ckks.Parameters, btpParams Parameters, btpKey Ev
 		bb.SlotsToCoeffsParameters.Scaling *= bb.params.DefaultScale().Float64() / (bb.evalModPoly.ScalingFactor().Float64() / bb.evalModPoly.MessageRatio())
 	}
 
-	bb.stcMatrices = advanced.NewHomomorphicEncodingMatrixFromLiteral(bb.SlotsToCoeffsParameters, encoder)
+	bb.stcMatrices = advanced.NewHomomorphicDFTMatrixFromLiteral(bb.SlotsToCoeffsParameters, encoder)
 
 	encoder = nil
```

Comment:
Is this for enhanced garbage collection or not needed ?

Reply:
Yes it is